app/policies/application_policy.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Default Policies
+class ApplicationPolicy
+  attr_reader :user, :record
```

Comment:
If you rename the `user` to `current_user`, you can remove all `attr_reader :current_user` from the other policies, as it will be inherited. Same could be done with `record`, but I think in this case it's better to be explicit.